sql/driver/driver.go

```diff
@@ -41,6 +41,7 @@ func (d *roachDriver) Open(dsn string) (driver.Conn, error) {
 	}
 	ctx := &base.Context{}
 	ctx.InitDefaults()
+	ctx.Insecure = u.Scheme == "http"
```

Comment:
nevermind, found it. L52 in this file should be calling newSender, not newHTTPSender. It bypasses the sender registration and does not set the context appropriately.